### PR TITLE
feat: added summary citation to collection detail.

### PR DESCRIPTION
**Pull request by MillenniumFalconMechanic**
### Reviewers
**Functional:** 
@tihuan

---

## Changes
- Add summary citation to collection detail page ([summary citation mock](https://www.figma.com/file/18a4QULVIfpFQhTn1FGh8a/Filtering?node-id=1327%3A8052), [default DOI mock](https://www.figma.com/file/18a4QULVIfpFQhTn1FGh8a/Filtering?node-id=1457%3A7365) if publication metadata not found for DOI).

### CODE REVIEW

File: frontend/src/common/queries/filter.ts [hunk 1]
```diff
@@ -377,6 +383,40 @@ function buildDatasetRow(
...
+ * @param publisherMetadata - Publication metadata of a collection.
+ */
+export function buildSummaryCitation(
+  publisherMetadata?: PublisherMetadata
+): string {
+  if (!publisherMetadata) {
+    return "";
+  }
+  const { authors, journal, published_year: publishedYear } = publisherMetadata;
+  const [firstAuthor] = authors;
```
Comment (tihuan) [inline frontend/src/common/queries/filter.ts:398]: just curious, is it possible that `firstAuthor` could be `undefined`? If so `firstAuthor.name` would throw a runtime error 😄 !

File: frontend/src/views/Collection/utils.tsx [hunk 2]
```diff
@@ -24,18 +24,75 @@ const LINK_ORDER: COLLECTION_LINK_TYPE[] = [
...
+/**
+ * @deprecated Remove once feature flag is removed (#1718).
  * Returns collection metadata in preferred order of display.
  * @param links - links associated with collection.
  * @param contactName
  * @param contactEmail
  * @returns Array of collection metadata in preferred order of display.
  */
-export function buildCollectionMetadataLinks(
+export function buildCollectionMetadataLinksDeprecated(
```
Comment (tihuan) [inline frontend/src/views/Collection/utils.tsx:90]: love the `deprecated` suffix 🔥 !

**Review by tihuan — APPROVED** [review]
LGTM!! Thanks so much for another great PR, @MillenniumFalconMechanic 🙏 🚀 ✨ !!

File: frontend/src/common/queries/filter.ts (hunk 1 above)
Comment (MillenniumFalconMechanic) [reply frontend/src/common/queries/filter.ts:398]: Good catch, Timmy! Let me add some handling just in case.

**Pull request merged**